apps/wallet/src/ui/app/redux/slices/sui-objects/Coin.ts

```diff
@@ -92,10 +92,11 @@ export class Coin {
     ): Promise<SuiTransactionResponse> {
         await signer.syncAccountState();
         const coin = await Coin.selectCoin(signer, coins, amount);
-        return await signer.transferObject({
-            objectId: coin,
+        return await signer.pay({
+            inputCoins: [coin],
+            recipients: [recipient],
+            amounts: [Number(amount)],
             gasBudget: DEFAULT_GAS_BUDGET_FOR_TRANSFER,
-            recipient: recipient,
         });
     }
 
```

Comment:
on L122 of this file, I replaced it with below for testing
```
console.log("Running Pay txn in transfer Sui");
return await signer.pay({
  inputCoins: [Coin.getID(coin)],
  recipients: [recipient],
  amounts: [Number(amount)],
  gasBudget: DEFAULT_GAS_BUDGET_FOR_TRANSFER,
});

```

Reply:
The receipt page did not load properly, but the transfer itself went thru, will leave it to frontend folks to handle that.